mohitahlawat2001/gallery-app
# feat: Add A Logo Of Gallery App

fixes #9 
What This PR Do:
I Am Adding  A New Logo For This Gallery App...


How Should It Be Tested:
It's Very Easy Just Open the App And See the Tab You See there is A small Present On the Tab Where the App is running
